content/home/examples/a-simple-component.js

```diff
@@ -2,13 +2,13 @@ class HelloMessage extends React.Component {
   render() {
     return (
       <div>
-        Hello {this.props.name}
+        Olá, {this.props.name}!
       </div>
     );
   }
 }
 
 ReactDOM.render(
-  <HelloMessage name="Taylor" />,
+  <HelloMessage name="Daniel" />,
```

Comment:
Só para deixar um nome mais pt-BR, mesmo. :D

Reply:
@gnuns

Eu to num impasse em relação à tradução dos exemplos. Entendo sua intenção, mas creio que tinha alguma questão em relação ao sync com o repo original.

Eu acho que se forem só as strings é de boa, mas gostaria de mais opiniões (@glaucia86 @cezaraugusto @eduardomoroni ) ?

Reply:
hahaha eu gostei da ideia, mas quanto mais o codigo original puder ser mantido, melhor. melhor deixar como o original.

nao ha nenhum impedimento forte para nao mudar mas abre precedente para outras mudancas, por exemplo alguem pode argumentar no futuro que `HelloMessage` deveria ser traduzido para `MensagemDeOi` "pra deixar mais pt-BR" e eu nao vou ter um argumento pra negar.

claro, se mudar pra "Cezar" eu deixo :P brincadeira